packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -42,7 +42,7 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
       (e) => {
         if (!disabled) {
           context?.onDismissMenu();
-          onClick ? onClick() : context.onItemClick(itemKey);
+          onClick ? onClick() : context?.onItemClick(itemKey);
```

Comment:
There can be an error that onItemClick is not a function if context is undefined

Reply:
I think we should be okay since CMContext is always created the ContextualMenu. onItemClick is by default an empty function unless overridden by the user. (file: ContextualMenu.tsx)